address.json

```diff
@@ -45,7 +45,8 @@
   "polkadot-airdrop.org": [
     "12We44NDCNUWa9tnfgxmSc38vDXFXcnLDkGdL2Xy45mnhZ52",
     "14BFcanA3zYTpbCmzKGXVEnFe9WoBCq3JdPQdnsiEgEhXSA2",
-    "14oEqqqCabUQdsdq1zP1qdb8b4uhEnc2mL1YYHKf8xYbh8FB"
+    "14oEqqqCabUQdsdq1zP1qdb8b4uhEnc2mL1YYHKf8xYbh8FB",
+    "12We44NDCNUWa9tnfgxmSc38vDXFXcnLDkGdL2Xy45mnhZ52"
```

Comment:
It is there, the first entry.

Reply:
Whoops, sorry I used ctrl-f and didn't seem to find it, maybe other tab was in focus, I do apologize!

Reply:
No issues, I really do appreciate it. I'd much rather say "oops" than not have it at all.